data/catalogue.csv

```diff
@@ -202,3 +202,5 @@ dateAdded,dateUpdated,url,name,description,documentation,license,maintainer,area
 2022-02-03,2022-02-03,https://admiraltyapi.portal.azure-api.net/products/tidal-api-premium,Tidal API - Premium,"The Tidal API provides an authoritative source of tidal height predictions for Standard and Secondary tidal level stations. Also, now available are Tidal Stream Events and Rates for a range of geographic locations.\n\nPremium provides access to historical and the current plus 1 years’ worth of tidal events and interval predictions for a wide range of tidal height stations and stream locations around the British Isles and Ireland. This includes The Isle of Man and The Channel Islands consisting of the Bailiwicks of Jersey and Guernsey.\n\n- 100 calls per second\n\n- 100,000 calls per month\n\n- 1 year subscription - £300 ex VAT",https://admiraltyapi.portal.azure-api.net/docs/services/uk-tidal-api-premium,,customerservices@ukho.gov.uk,The British Isles and Ireland,,,uk-hydrographic-office
+2022-02-15,2022-02-15,https://data.api.trade.gov.uk/v1/datasets/uk-tariff-2021-01-01/versions/latest/tables/commodities/data?format=csv&download,UK Global Tariff (commodities),"The UK Global Tariff (UKGT) is the UK's first independent tariff policy which replaced the EU Common External Tariff (CET) which applied until 31 December 2020. This tariff entered into force on 1 January 2021. The UKGT applies to all goods imported into the UK. It lists preferential measures where the UK has entered into a new trade agreement or arrangement with a third country or territory. For other countries and territories, it shows the UK's Most Favoured Nation (MFN) tariffs. The dataset does not include other import duties (such as VAT) and details of quota volumes.,,This table (one of 3 available) shows all of the commodities that are part of the UK's goods classification and their English descriptions.,,Codes are organised in a hierarchy with each ""parent"" code having zero or more ""child"" codes that represent a more specific classification of the parent commodity.,,Commodities at all depths of the commodity hierarchy are included, but only commodities without any children codes are declarable. The current parent code of each commodity code is also listed and is accurate at the time the file was generated – note that parent codes can change over time.,,The tariffs in this dataset may be a representation of future events and as such are subject to change.,,You can use this data when you need to make a calculation against the new duties that apply in the UK. You cannot use this data to calculate the total cost of an import because it does not include VAT.",https://data.api.trade.gov.uk/v1/datasets/uk-tariff-2021-01-01/versions/v2.2.0/metadata?format=html,https://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/
```

Comment:
Needs a bit of reformatting to replace extra commas with newlines in the description and add the organisation:

```suggestion
2022-02-15,2022-02-15,https://data.api.trade.gov.uk/v1/datasets/uk-tariff-2021-01-01/versions/latest/tables/commodities/data?format=csv&download,UK Global Tariff (commodities),"The UK Global Tariff (UKGT) is the UK's first independent tariff policy which replaced the EU Common External Tariff (CET) which applied until 31 December 2020. This tariff entered into force on 1 January 2021. The UKGT applies to all goods imported into the UK. It lists preferential measures where the UK has entered into a new trade agreement or arrangement with a third country or territory. For other countries and territories, it shows the UK's Most Favoured Nation (MFN) tariffs. The dataset does not include other import duties (such as VAT) and details of quota volumes. This table (one of 3 available) shows all of the commodities that are part of the UK's goods classification and their English descriptions.\n\n Codes are organised in a hierarchy with each ""parent"" code having zero or more ""child"" codes that represent a more specific classification of the parent commodity. \n\nCommodities at all depths of the commodity hierarchy are included, but only commodities without any children codes are declarable. The current parent code of each commodity code is also listed and is accurate at the time the file was generated – note that parent codes can change over time. \n\nThe tariffs in this dataset may be a representation of future events and as such are subject to change. \n\nYou can use this data when you need to make a calculation against the new duties that apply in the UK. You cannot use this data to calculate the total cost of an import because it does not include VAT.",https://data.api.trade.gov.uk/v1/datasets/uk-tariff-2021-01-01/versions/v2.2.0/metadata?format=html,https://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/,,,,,department-for-international-trade
```

Reply:
Ah, thank you - I wasn't sure what the convention was for csv strings